src/components/Header&Footer/Footer.css

```diff
@@ -0,0 +1,21 @@
+.footer{
+  width: 100%;
+  height: 10%;
+  background-color: pink;
```

Comment:
To get full score on accessibility you should have a bigger contrast between the color of the text and background :)